src/org/usfirst/frc/team2335/robot/subsystems/Climb.java

```diff
@@ -0,0 +1,38 @@
+package org.usfirst.frc.team2335.robot.subsystems;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.Victor;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+
+public class Climb extends Subsystem 
+{
+    //Motor for climbing
+	Victor climbMotor;
+	
+	public Climb()
+	{
+		super();
```

Comment:
I'm not sure, but it's there when you create a new subsystem file, so I thought it would be best to leave it.